lib/kadai-core-test/src/test/java/acceptance/PojoTest.java

```diff
@@ -45,26 +44,26 @@
 /** check classes with a custom equals and hashcode implementation for correctness. */
 class PojoTest {
 
-  private static final List<Class<?>> POJO_CLASSES = getPojoClasses();
-
-  private static List<Class<?>> getPojoClasses() {
-    // TODO how to identify pojos? Is overwritten equals method enough?
```

Comment:
Why did you delete the TODO?

Reply:
Is it ever going to be taken care of? :)